src/main/java/heronarts/lx/dmx/DmxSplitModulator.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2023- Justin K. Belcher, Mark C. Slee, Heron Arts LLC
+ *
+ * This file is part of the LX Studio software library. By using
+ * LX, you agree to the terms of the LX Studio Software License
+ * and Distribution Agreement, available at: http://lx.studio/license
+ *
+ * Please note that the LX license is not open-source. The license
+ * allows for free, non-commercial use.
+ *
+ * HERON ARTS MAKES NO WARRANTY, EXPRESS, IMPLIED, STATUTORY, OR
+ * OTHERWISE, AND SPECIFICALLY DISCLAIMS ANY WARRANTY OF
+ * MERCHANTABILITY, NON-INFRINGEMENT, OR FITNESS FOR A PARTICULAR
+ * PURPOSE, WITH RESPECT TO THE SOFTWARE.
+ *
+ * @author Mark C. Slee <mark@heronarts.com>
+ * @author Justin K. Belcher <justin@jkb.studio>
+ */
+
+package heronarts.lx.dmx;
+
+import heronarts.lx.LXCategory;
+import heronarts.lx.modulator.LXModulator;
+import heronarts.lx.parameter.BooleanParameter;
+import heronarts.lx.parameter.CompoundParameter;
+
+/**
+ * Modulator receiving two ranges of input from one DMX channel.
+ * Input     Output
+ *  0        Output1 = 0    Output2 = 0
+ *  1-127    Output1 = 0-1  Output2 = 0
+ *  128-255  Output1 = 0    Output2 = 0-1
+ *
+ * @author Justin K. Belcher
+ */
+@LXModulator.Global("DMX Split")
+@LXModulator.Device("DMX Split")
+@LXCategory(LXCategory.DMX)
+public class DmxSplitModulator extends DmxModulator {
```

Comment:
This is pretty strange, is this a common hardware behavior? What does this?

This whole thing is kind of giving me the heebeegeebees... also wondering if the utility math stuff like `Operator` modulator can't just be used to implement this behavior.

Is this a common thing?

Reply:
In retrospect this was built prior to the DmxRangeModulator, and is really just two DmxRangeModulators without being as flexible.  Removed!